org.freedesktop.Sdk.Extension.texlive.yml

```diff
@@ -56,30 +60,47 @@ modules:
       - type: archive
         url: https://pi.kwarc.info/historic/systems/texlive/2021/texlive-20210325-source.tar.xz
         sha256: 7aefd96608d72061970f2d73f275be5648ea8ae815af073016d3106acc0d584b
+        x-checker-data:
+          type: html
+          url: https://pi.kwarc.info/historic/systems/texlive/2021/
+          version-pattern: texlive-([\d\.]+).source.tar.xz
+          url-template: https://pi.kwarc.info/historic/systems/texlive/2021/texlive-$version-source.tar.xz
```

Comment:
For the next (2022) release the url will probably change to something like `.../texlive/2022/texlive-$version-source.tar.xy` (at least from looking at the last years). Will be this covered by the `x-checker-data`?